FF/Velocity.C

```diff
@@ -76,7 +112,7 @@ void preciceAdapter::FF::Velocity::read(double* buffer, const unsigned int dim)
 
 bool preciceAdapter::FF::Velocity::isLocationTypeSupported(const bool meshConnectivity) const
 {
-    return (this->locationType_ == LocationType::faceCenters);
+    return (this->locationType_ == LocationType::faceCenters || this->locationType_ == LocationType::volume);
```

Comment:
This is about supporting mesh connectivity. Do we support mesh connectivity for volume meshes?

Reply:
What I meant in this comment was not to implement mesh connectivity as part of this PR (even though it would be nice, but let's document it in an issue first). I just meant to remove this addition, because we don't support it yet.

Reply:
Your original suggestion was in the right direction. I got confused, as the connectivity feature was added on top of everything much later, but the architecture did not really allow us for a nice implementation.

If we add the `|| this->locationType_ == LocationType::volume`, this means that volume centers are supported regardless of mesh connectivity or not. However, we need to distinguish here between mesh connectivity (supporting only `faceCenters`) and no mesh connectivity (supporting both).

As you already pointed out in a direct message, we can do this similarly to the Temperature class.